web/src/features/prompts/components/prompt-detail.tsx

```diff
@@ -52,53 +70,74 @@ export const PromptDetail = (props: PromptDetailProps) => {
+                href: `/project/${projectId}/prompts/`,
               },
               {
                 name: prompt.name,
-                href: `/project/${props.projectId}/prompts/${encodeURIComponent(props.promptName)}`,
+                href: `/project/${projectId}/prompts/${encodeURIComponent(promptName)}`,
               },
               { name: `Version ${prompt.version}` },
             ]}
             actionButtons={
               <>
                 <PromotePrompt
-                  projectId={props.projectId}
+                  projectId={projectId}
                   promptId={prompt.id}
                   promptName={prompt.name}
                   disabled={prompt.isActive}
                   variant="outline"
                 />
-                <CreatePromptDialog
-                  projectId={props.projectId}
-                  title="Update Prompt"
-                  subtitle="We do not update prompts, instead we create a new version of the prompt."
-                  promptName={prompt.name}
-                  promptText={prompt.prompt}
-                  promptConfig={jsonSchema.parse(prompt.config)}
+
+                {isPlaygroundEnabled ? (
+                  <Link
+                    href={`/project/${projectId}/playground?promptId=${encodeURIComponent(prompt.id)}`}
+                  >
+                    <Button
+                      variant="outline"
+                      title="Test in prompt playground"
+                      size="icon"
+                    >
+                      <Terminal className="h-5 w-5" />
+                    </Button>
+                  </Link>
+                ) : null}
+
+                <Link
+                  href={`/project/${projectId}/prompts/new?promptId=${encodeURIComponent(prompt.id)}`}
                 >
                   <Button variant="outline" size="icon">
                     <Pencil className="h-5 w-5" />
                   </Button>
-                </CreatePromptDialog>
+                </Link>
+
                 <DeletePromptVersion
-                  projectId={props.projectId}
+                  projectId={projectId}
                   promptVersionId={prompt.id}
                   version={prompt.version}
                   countVersions={promptHistory.data.length}
                 />
                 <DetailPageNav
                   key="nav"
-                  currentId={props.promptName}
-                  path={(name) => `/project/${props.projectId}/prompts/${name}`}
+                  currentId={promptName}
+                  path={(name) => `/project/${projectId}/prompts/${name}`}
                   listKey="prompts"
                 />
               </>
             }
           />
         </div>
         <div className="col-span-2 md:h-full">
-          <CodeView content={prompt.prompt} title="Prompt" />
+          <div className="mx-auto mb-5 w-full rounded-lg border text-base leading-7">
```

Comment:
Maybe we can move the badge right next to the prompt name? I think the UI is not very nice right now.